test/system/entities_test.rb

```diff
@@ -0,0 +1,45 @@
+require 'application_system_test_case'
+
+class EntitiesTest < ApplicationSystemTestCase
+  setup do
+    @entity = entities(:one)
+  end
+
+  test 'visiting the index' do
+    visit entities_url
+    assert_selector 'h1', text: 'Entities'
+  end
+
+  test 'should create entity' do
+    visit entities_url
+    click_on 'New entity'
+
```

Comment:
* When i run the tests i get errors, kindly ensure that your tests are passing and update your readme with steps to run your app properly 
![image](https://github.com/alexmedinasf/budget_app/assets/52497246/335825bf-3f74-4bc8-b418-1d324f2e0a8f)

Reply:
hey @Davidosky007 if its possible can you explain to me how to fix this test errors?